### PR TITLE
replaced unicode arrows with template SVGs

### LINKED ISSUE #1557: Bug: unicode icons within button icons are not centered properly
### Current Behavior

The first/last icons are not centered in the button itself, as can be seen in the following picture:
Screenshot 2023-05-24 at 17 41 18

This is not a major problem, since the `buttonTextFirst` and `buttonTextLast` props can be set to override this.

However, it could maybe be nice to have a (close to) pixel perfect look out of the box!

Right now, the following HTML-encoded characters are used:
```typescript
/** Set the text label for the Previous button. */
export let buttonTextPrevious: CssClasses = '&larr;';
/** Set the text label for the Next button. */
export let buttonTextNext: CssClasses = '&rarr;';
/** Set the text label for the First button. */
export let buttonTextFirst: CssClasses = '&laquo;';
/** Set the text label for the Last button. */
export let buttonTextLast: CssClasses = '&raquo;';
```

I tried to find better alternatives but without luck. So maybe additional CSS could be needed? Or some SVG embeds? Not sure!

### Reproduction / Steps To Reproduce

Just use the Paginator component with the default values and you should spot the problem.

Tested on Safari and Chrome

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by Mahmoud-zino**
## Linked Issue

Closes #1557

## Description

replaced Unicode arrows with templated SVGs.

## Note
[chevrons-left](https://fontawesome.com/icons/chevrons-left?f=classic&s=solid) & [chevrons-right](https://fontawesome.com/icons/chevrons-right?f=classic&s=solid) Icons are pro icons so I ended up using [angles-right](https://fontawesome.com/icons/angles-right?f=classic&s=solid) and [angles-left](https://fontawesome.com/icons/angles-left?f=classic&s=solid) Icons which looks the same.

## Changsets

bugfix: replaced unicode arrows with template SVGs in paginator

## Checklist

Please read and apply all [contribution requirements](https://www.skeleton.dev/docs/contributing).

- [x] This PR targets the `dev` branch (NEVER `master`)
- [x] Documentation reflects all relevant changes
- [x] Branch is prefixed with: `docs/`, `feat/`, `chore/`, `bugfix/`
- [x] Ensure Svelte and Typescript linting is current - run `pnpm check`
- [x] Ensure Prettier linting is current - run `pnpm format`
- [x] All test cases are passing - run `pnpm test`
- [x] Includes a changeset (if relevant; see above)

### CODE REVIEW

**Comment by endigo9740** [comment]
@Mahmoud-zino those actually look really sharp - one addition note, we'll need to adjust for not just text color, but fill color as well when we implement the SVG so that these are still visible:

image

**Comment by Mahmoud-zino** [comment]
@endigo9740 I added the `fill-current` to the `buttonClasses` so if the user wants to change it they can do it easily.
If this is not correct please tell me and I will change it.

File: packages/skeleton/src/lib/components/Paginator/Paginator.svelte [hunk 1]
```diff
@@ -150,7 +163,7 @@
 		{#if showFirstLastButtons}
 			<button
 				type="button"
-				class={buttonClasses}
+				class="{buttonClasses} 0"
```
Comment (endigo9740) [inline packages/skeleton/src/lib/components/Paginator/Paginator.svelte:166]: Was this a typo?

Comment (Mahmoud-zino) [reply packages/skeleton/src/lib/components/Paginator/Paginator.svelte:166]: ops, yeah sorry this was a typo, I will fix it.

**Pull request merged**